components/sidebar-layout.tsx

```diff
@@ -129,7 +129,7 @@ const SidebarLayout = ({ children }: Props) => {
   const workflowsNavigation = workflowItems(router);
 
   return (
-    <div className="h-screen w-screen bg-white">
+    <div className="h-screen w-screen bg-white overflow-hidden">
```

Comment:
What's this for? Does this affect the other views?

Reply:
The scrolling was happening at the highest tier parent, this div. I needed to disable it here and set it on dynamic so the modal height will be the same as the page height.

Edited to add overflow to pages that need it

Reply:
Visually that looks odd to me in your video, the header staying still vs the whole page scrolling. Playing with overflow is likely an anti pattern, I think there's something different you want here